### PR TITLE
standardize child age calculation for announcements; update announcement template

### PR DESCRIPTION
**Pull request by kimberscott**
Sorry for not catching this earlier - we define a year as 365 days throughout, so to avoid people getting notifications when they "age in" but then being told they're not eligible when they get to Lookit (which would be the case for every study for children at least 1 year old), use that definition here too.

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -657,16 +659,29 @@ def send_as_email(self):
         self.save()
 
 
-def create_subject_for_study_notification(study, children):
-    latter_half = f' invited to take part in "{study.name}" on Lookit!'
+def create_string_listing_children(children):
     child_names = [child.given_name for child in children]
+    num_children = len(child_names)
```
Comment (kimberscott) [inline accounts/models.py:664]: apologies for not using `:=` here - precommit was balking at it even as already used?

Comment (Datamance) [reply accounts/models.py:664]: How were you trying to use it?

Comment (Datamance) [reply accounts/models.py:664]: oh I see what you're saying - yeah, that's strange. If our precommit environments are identical that shouldn't happen, unless something actually changed in the code itself that would invalidate the syntax.

Comment (Datamance) [reply accounts/models.py:664]: on that note - make sure your precommit and virtualenv are both using 3.8.x (I think our pipenv env will force 3.8.3 on you)

**Pull request merged**